documentation/modules/configuring/con-config-kafka-connector.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='con-kafka-connector-config-{context}']
+= Configuring Kafka Connect connectors
+
+[role="_abstract"]
+The `KafkaConnector` resource provides a Kubernetes-native approach to management of connectors by the Cluster Operator.
+To create, delete, or reconfigure connectors with `KafkaConnector` resources, you must set the `use-connector-resources` annotation to `true`  in your `KafkaConnect` custom resource.
+
+.Annotation to enable KafkaConnectors
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaConnectApiVersion}
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true"
+  # ...
+----
+
+When the use-connector-resources annotation is enabled in your KafkaConnect configuration, you must define and manage connectors using `KafkaConnector` resources. These resources must be deployed in the same namespace as the Kafka Connect cluster they are associated with, and the Kafka cluster interacting with the external data system.
+
+NOTE: Alternatively, you can manage connectors using the Kafka Connect REST API instead of `KafkaConnector` resources.
+To use the API, you must remove the annotation to use `KafkaConnector` resources in the `KafkaConnect` the resource.
```

Comment:
Maybe this way it would be a bit more clear?

```suggestion
To use the API, you must remove the `strimzi.io/use-connector-resources` annotation in the `KafkaConnect` the resource.
```